keras/applications/regnet.py

```diff
@@ -0,0 +1,1285 @@
+  
+  Args:
+    x: Input tensor. Should be 224x224, rescaled and normalized to [0,1].   
+
+  Returns:
+    Output tensor of the Stem
+  """
+  x = layers.Conv2D(32, (3, 3), strides=2, use_bias=False)(x)
+  x = layers.BatchNormalization(momentum=0.9, epsilon=1e-5)(x)
+  x = layers.ReLU()(x)
+
+  return x
+
+
+def XBlock(inputs,
+           filters_in,
+           filters_out,
+           group_width,
+           stride=1):
+  """
+  Implementation of X Block. 
+  Reference: [Designing Network Design Spaces](https://arxiv.org/abs/2003.13678)
+
+  Args:
+    inputs: input tensor
+    filters_in: Filters in the input tensor
+    filters_out: Filters in the output tensor
+    group_width: Group width
+    stride: Stride
+
+  Return:
+    Output tensor of the block 
+  """
+
+  if filters_in != filters_out and stride == 1:
+    raise ValueError("""Input filters and output filters are not equal for stride
+                         1. Please check inputs and try again.""")
+
+  # Declare layers
+  groups = filters_out // group_width
+
+  relu1x1 = layers.ReLU()
+  relu3x3 = layers.ReLU()
+  relu = layers.ReLU()
+  
+  if stride != 1:
+    skip = layers.Conv2D(filters_out, (1,1), strides=stride, use_bias=False)(inputs)
+    skip = layers.BatchNormalization(momentum=0.9, epsilon=1e-5)(skip)
+    conv_3x3 = layers.Conv2D(filters_out, (3, 3), use_bias=False, strides=stride,
+        groups=groups)
+  else:
+    skip = inputs
+    conv_3x3 = layers.Conv2D(filters_out, (3, 3), use_bias=False,
+                             groups=groups)
+  
+  # Build block
+  # conv_1x1_1
+  x = layers.Conv2D(filters_out, (1,1), use_bias=False)(inputs)
+  x = layers.BatchNormalization(momentum=0.9, epsilon=1e-5)(x)
+  x = relu1x1(x)
+  
+  # conv_3x3
+  x = conv_3x3(x)
+  x = layers.BatchNormalization(momentum=0.9, epsilon=1e-5)(x)
+  x = relu3x3(x)
+  
+  # conv_1x1_2
+  x = layers.Conv2D(filters_out, (1, 1), use_bias=False)(x)
+  x = layers.BatchNormalization(momentum=0.9, epsilon=1e-5)(x)
+
+  x = relu(x + skip)
+
+  return x
+
+
+def YBlock(inputs,
+           filters_in,
+           filters_out,
+           group_width,
+           stride=1,
+           se_ratio=0.25):
+  """
+  Implementation of Y Block. 
+  Reference: [Designing Network Design Spaces](https://arxiv.org/abs/2003.13678)
+
+  Args:
+    inputs: input tensor
+    filters_in: Filters in the input tensor
+    filters_out: Filters in the output tensor
+    group_width: Group width
+    stride: Stride
+    se_ratio: Expansion ration for Squeeze and Excite block
+
+  Return:
+    Output tensor of the block 
+  """
+
+  if filters_in != filters_out and stride == 1:
+    raise ValueError("""Input filters and output filters are not equal for stride
+                         1. Please check inputs and try again.""")
+
+  groups = filters_out // group_width
+  se_filters = int(filters_out * se_ratio)
+
+  relu1x1 = layers.ReLU()
+  relu3x3 = layers.ReLU()
+  relu = layers.ReLU()
```

Comment:
Instead of initializing like this prefer to have them inline i.e. `x = layers.ReLU()(x)` until and unless you have a good reason to do otherwise. Also, ensure all the layers are properly named.